google/cloud/storage/client.py

```diff
@@ -836,6 +841,126 @@ def get_hmac_key_metadata(
         metadata.reload(timeout=timeout)  # raises NotFound for missing key
         return metadata
 
+    def get_signed_policy_v4(
```

Comment:
Pending final confirmation on naming, leaving this as an open item.

Reply:
Done, working now on conformance tests